test/Microsoft.AspNetCore.Razor.Design.Test/IntegrationTests/BuildServerIntegrationTest.cs

```diff
@@ -159,5 +164,31 @@ public Task Build_SimpleMvc_WithServer_UsingDesktopMSBuild_CanBuildSuccessfully(
             Assert.Equal(0, exitCode);
             Assert.Contains("shut down completed", output.ToString());
         }
+
+        [Fact]
+        [InitializeTestProject("SimpleMvc")]
+        public void ServerStartup_WritesProcessAndPipeToFile()
+        {
+            // Arrange & Act
+            
+            // We expect the server to have started by now.
+
+            var processId = _buildServer.ProcessId;
+            var expectedFileName = $"rzc-{processId}";
+            var homeEnvVariable = PlatformInformation.IsWindows ? "USERPROFILE" : "HOME";
+            var path = Path.Combine(Environment.GetEnvironmentVariable(homeEnvVariable), ".dotnet", "pids", "build", expectedFileName);
+
+            // Assert
+            Assert.True(File.Exists(path));
```

Comment:
Didn't you say this had timing issues?

Reply:
Yes. I am currently working on unit tests and will removing this in the next update.